client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -313,7 +313,7 @@ private ConcurrentHashMap<Integer, PartitionLocation> registerShuffle(
     return registerShuffleInternal(
         shuffleId,
         numMappers,
-        numMappers,
```

Comment:
Good catch!  Although num partitions is actually correctly used when register shuffle

Reply:
Yes, currently this parameter is only used in printing logs